### PR TITLE
Added a qtile profile

### PR DESCRIPTION
**Pull request by edl2**
Working on adding a qtile profile, should nitrogen be included and nano vs vim and dmenu vs rofi should be discussed.

### CODE REVIEW

**Comment by Torxed** [comment]
Looks sensible to me. Looking forward to testing a python based WM :)

**Comment by dylanmtaylor** [comment]
> Looks sensible to me. Looking forward to testing a python based WM :)

I'm not really sure we want too many more desktop environment options - especially if it's somewhat opinionated by us, for example, by picking things like vim, nano, and nitrogen. We should try to ship upstream without packages like these. Also, vim/nano is redundant with base packages, IMO. They don't really belong in the desktop profile.

File: profiles/qtile.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+# New way of defining packages for a profile, which is iterable and can be used out side
+# of the profile to get a list of "what packages will be installed".
+__packages__ = [
+	'qtile',
+	'alacritty',
+	'lightdm-gtk-greeter',
+	'lightdm',
+	'dmenu',
+	'pcmanfm',
+	'vim',
```
Comment (dylanmtaylor) [inline profiles/qtile.py:16]: remove vim and nano, these aren't DE specific and don't belong in the desktop profile.

File: profiles/qtile.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	'nano',
+	'nitrogen',
+
```
Comment (dylanmtaylor) [inline profiles/qtile.py:19]: delete blank spaces here for linter, and remove trailing comman from nitrogen. Also, is nitrogen really needed?

File: profiles/qtile.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+			return imported._prep_function()
+		else:
+			print('Deprecated (??): xorg profile has no _prep_function() anymore')
+
+
+if __name__ == 'qtile':
+
+
+	# Install dependency profiles
+	archinstall.storage['installation_session'].install_profile('xorg')
```
Comment (dylanmtaylor) [inline profiles/qtile.py:47]: qtile looks like it can run with Wayland, maybe this isn't needed?

**Comment by dylanmtaylor** [comment]
Please correct the flake8 errors - only a few of them, so shouldn't take too long. https://github.com/archlinux/archinstall/runs/4999156240?check_suite_focus=true

File: profiles/qtile.py (hunk 2 above)
Comment (edl2) [reply profiles/qtile.py:15]: For linter and the flake errors do I remove all blank spaces ?

Comment (dylanmtaylor) [reply profiles/qtile.py:15]: Just the blank lines where it's complaining.

**Comment by dylanmtaylor** [comment]
Hmm, on second thought, we might want to do what we do in GNOME and install Xorg here, for now: https://github.com/archlinux/archinstall/blob/master/profiles/gnome.py#L24

**Comment by edl2** [comment]
keep in mind that reverting 34e89b1 would also revert some spacing error fixes

**Comment by tuhanayim** [comment]
You should add this profile to `profiles/desktop.py`.
https://github.com/archlinux/archinstall/blob/master/profiles/desktop.py#L23

**Comment by dylanmtaylor** [comment]
> keep in mind that reverting 34e89b1 would also revert some spacing error fixes

Ah. In that case, maybe just copy the block out of the history. And the other commentator is right, it needs to be in desktop.py for it to show in the list.

**Comment by Torxed** [comment]
> Hmm, on second thought, we might want to do what we do in GNOME and install Xorg here, for now: https://github.com/archlinux/archinstall/blob/master/profiles/gnome.py#L24
> 
> This would just be a revert of [34e87b1](https://github.com/archlinux/archinstall/commit/34e87b13accfba16cd8a145e513f225de7fd86fc)
> 
> @Torxed , thoughts?

I agree on this one.
I revert the commit-hash in question, since this is the first time committing to a project - in case git is new too here's how I did it:
```
cd archinstall
git revert 34e87b13accfba16cd8a145e513f225de7fd86fc
# modify/fix the merge issue it complained about with sublime
git add profiles/qtile.py
git revert --continue
# Used the commit message as-is
git push
```

**Comment by Torxed** [comment]
> You should add this profile to `profiles/desktop.py`. https://github.com/archlinux/archinstall/blob/master/profiles/desktop.py#L23

Added it to the desktop selection :) Nice to point out where to add it to new contributors, helps a lot!

I think this PR is ready to go in.

**Pull request merged**